### DISCUSSION TITLE
Question about Earth2Java compatibility

### DISCUSSION THREAD

**Discussion by vico93** [FrozenBlock/WilderWild#292] [Q&A] [answered]
I noticed a `Custom flower generation` option on the mod options. Does it affects other mods flower generation?

Currently i use [Earth2Java](https://github.com/Slexom/earth2java), and it add some custom flowers, and i'm afraid to lose generation of those if i enable that option.

**Comment by EthanRStokes** [comment, marked as answer]
no, our configs should not change anything in other mods
